lib/graph3d/Graph3d.js

```diff
@@ -82,13 +82,15 @@ Graph3d.DEFAULTS = {
       line    : {
         height        : '40px',
         width         : '0',
-        borderLeft    : '1px solid #4d4d4d'
+        borderLeft    : '1px solid #4d4d4d',
+        pointerEvents : 'none'
```

Comment:
What's this?

Reply:
when tooltip is enabled, if you do ctrl+wheel, the event is on the dot/line of the tooltip and not on the graph, so the browser is zooming but not the graph. Pointer events is here to send the event through the html element to the graph